src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -792,6 +797,23 @@ public WorkspacePanel(final MCreator mcreator) {
 
 		duplicateElement.addActionListener(e -> duplicateCurrentlySelectedModElement());
 
+		searchElement.setIcon(UIRES.get("16px.search"));
+		searchElement.addActionListener(e -> {
+			if (list.getSelectedValuesList().stream().anyMatch(i -> i instanceof ModElement)) {

```

Comment:
Here you can hardcode check if i getType is TAG and instead of calling searchModElementUsages and all code below, just show a simple option pane. This will be much clearer than help tip which does not apply to any other case

Reply:
So tag elements should be ignored?

Reply:
Maybe do like this?
```java
		boolean tagsSelected;
<...>
		if (el.getType() == ModElementType.TAG)
			tagsSelected = true;
<...>
		if (tagsSelected) {
			JOptionPane.showMessageDialog(mcreator,
					"Some selected elements can be weakly referenced by other mods. Only references by elements in this project will be shown.",
					"Weak references", JOptionPane.WARNING_DIALOG);
		}
```

Reply:
No.

I would be explicit about tags in the message as this applies to tags specially and only. And not invoke reference finder. As explained here:

![image](https://github.com/MCreator/MCreator/assets/16374228/20ef571c-c79e-4977-aaec-546e2b83c15d)

![image](https://github.com/MCreator/MCreator/assets/16374228/8dc7793f-f02b-462e-af54-12e7753d52b8)

Reply:
Did I get it?

Reply:
I think you did it well, also handled ok the multi-ME-selection :)